src/util/pixelSetter.js

```diff
@@ -1,5 +1,12 @@
+/**  
+ * @param {number} x x-coordinate.
+ * @param {number} y y-coordinate.
+ * @param {object} value
```

Comment:
value is an array, `[r, g, b, a]` :)

Reply:
@HarshKhandeparkar ,made the changes. Thanks :)